sw/host/opentitanlib/src/chip/earlgrey.rs

```diff
@@ -2,325 +2,327 @@
 // Licensed under the Apache License, Version 2.0, see LICENSE for details.
 // SPDX-License-Identifier: Apache-2.0
 
+#![allow(dead_code)]
```

Comment:
is this file autogenerated?

If so, a file header comment may make that more clear.

Reply:
That sound reasonable. I will add headers to the files with information 
that the files were generated automatically.

Reply:
@cfrantz What do you think about adding a sub-folder

Reply:
Header added. In the future we should add timestamp and git sha.
Also it will be good to rewrite topgen is such way that it can be invoked similar to reggen.